src/figuresClasses.ts

```diff
@@ -1,11 +1,111 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  getArea(): number {
+    const aSide = this.a;
+    const bSide = this.b;
+    const cSide = this.c;
```

Comment:
[CODE STYLE] - The `getArea` function doesn't need to accept any parameters as per checklist item 1. All necessary values are already available as object fields. You don't need to declare local variables `aSide`, `bSide`, and `cSide` since you can directly use `this.a`, `this.b`, and `this.c`.